Update antimalware-procedures.md (#2994)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer/antispam-and-antimalware/antimalware-protection/antimalware-procedures.md

- Configure malware filtering to scan messages that were already scanned by Exchange Online Protection (EOP).

> - Configure a Malware Filtering  Bypass for a Recipient or Group of Recipients

Reviewer: ```suggestion
- Configure a malware filtering  bypass for a recipient or group of recipients
```

```PowerShell
Get-MalwareFilteringServer | Format-List Name, ForceRescan
```

> ## Configure a Malware Filter Bypass for a recipient or Group of Recipients

Reviewer: ```suggestion
## Configure a malware filtering bypass for a recipient or group of recipients
```

> To allow for a particular recipient or group of recipents to recieve emails with attachments that would otherwise be detected and thus deleted by the default anti-Malware Policy, a new anti-Malware Policy has to be created, either using EAC or Porweshell. This policy shoud be set to Scan emails for all recipients and an  exclusion condition must be set within that policy for the Recipient or Group that is intended to recieve such content.

Reviewer: ```suggestion
To allow for a particular recipient or group of recipients to receive emails with attachments that would otherwise be detected and thus deleted by the default anti-malware policy, a new anti-malware policy has to be created, either using EAC or PowerShell. This policy should be set to scan emails for all recipients and an exclusion condition must be set within that policy for the recipient or group that is intended to receive such content.
```

> Using EAC to Creeate the new anti-Malware Policy:

Reviewer: ```suggestion
Using EAC to create the new anti-malware policy:
```

![image](https://user-images.githubusercontent.com/98214653/154773999-f5556f76-04a6-4be4-a9f4-90726ec886e3.png)

![image](https://user-images.githubusercontent.com/98214653/154774156-8c835de2-892b-40c1-85d1-5388dfbceeef.png)

> This will create a Malware Policy that will scan all email traffic exepct messages sent to the excempt user or group.

Reviewer: ```suggestion
This will create a malware policy that will scan all email traffic except messages sent to the exempt user or group.
```

> Using Exchange Management Shell to create an anti-Malware Policy:

Reviewer: ```suggestion
Using Exchange Management Shell to create an anti-malware policy:
```

> [Use the Exchange Management Shell to modify a malware filter rule](/Exchange/antispam-and-antimalware/antimalware-protection/antimalware-procedures?view=exchserver-2019#use-the-exchange-management-shell-to-create-antimalware-policies)..

Reviewer: ```suggestion
[Use the Exchange Management Shell to modify a malware filtering rule](/Exchange/antispam-and-antimalware/antimalware-protection/antimalware-procedures?view=exchserver-2019#use-the-exchange-management-shell-to-create-antimalware-policies).
```

> 
> https://docs.microsoft.com/en-us/Exchange/antispam-and-antimalware/antimalware-protection/antimalware-procedures?view=exchserver-2019#use-the-exchange-management-shell-to-create-antimalware-policies

Reviewer: ```suggestion
```

> After creating the anti-Malware policy and setting the apropriate exclusion, a new anti-Malware policy has to be created in order for the unscaned messages to bypass the default anti-Malware Policy that always has the lowest priority. This change can achieved by using Exchange Management Shell to set the value of the paramenter "-BypassInboundMessages" of this anti-Malware Filter Rule to True. For detailed syntax and parameter information, see [Set-MalwareFilterPolicy](/powershell/module/exchange/set-malwarefilterpolicy?view=exchange-ps)..

Reviewer: ```suggestion
After creating the anti-malware policy and setting the appropriate exclusion, a new anti-malware policy has to be created in order for the unscanned messages to bypass the default anti-malware policy that always has the lowest priority. This change can be achieved by using Exchange Management Shell to set the value of the parameter "-BypassInboundMessages" of this anti-malware filtering rule to True. For detailed syntax and parameter information, see [Set-MalwareFilterPolicy](/powershell/module/exchange/set-malwarefilterpolicy?view=exchange-ps).
```

```PowerShell
Set-MalwareFilterPolicy -Identity "<PolicyName>" -BypassInboundMessages $true
```

> This method does not require service restart and can allow for Exchange Administrators and Security teams to temporarly or permanently bypass anti-Malware detection for a particular recipient or group, and thus having otherwise undeliverable messages delivered to that particular subgroup without compromising the security of the entire Exchange Organization and ultimately retaining security within their systems and for their clients.  

Reviewer: ```suggestion
This method does not require a service restart and can allow for Exchange administrators and security teams to temporarily or permanently bypass anti-malware detection for a particular recipient or group, thus having otherwise undeliverable messages delivered to that particular subgroup without compromising the security of the entire Exchange organization and ultimately retaining security within their systems and for their clients.  
```
